docs/_static/documentation_options.js

```diff
@@ -1,10 +1,11 @@
 var DOCUMENTATION_OPTIONS = {
     URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
-    VERSION: '2.4.0',
+    VERSION: '2.2.0+217.geefab596.dirty',
```

Comment:
Still seeing the `dirty` version, but this we be cleared up with the next, push, correct? 

@martinfleis

Reply:
@sjsrey you need to merge master into this branch and run sphinx again to get the correct version.

Reply:
i think this is just a remnant from the last time the docs were build locally on serges machine, this gets overwritten in the gh-pages branch when the docs are built as part of GHA